polyfill/test/zoneddatetime.mjs

```diff
@@ -730,6 +730,23 @@ describe('ZonedDateTime', () => {
     });
   });
 
+  describe('Maths', () => {
+    const earlier = ZonedDateTime.from('1976-11-18T15:23:30.123456789-03:00[America/Santiago]');
+    const later = ZonedDateTime.from('2019-10-29T10:46:38.271986102-03:00[America/Santiago]');
+    ['years', 'months', 'weeks', 'days', 'hours', 'minutes', 'seconds'].forEach((largestUnit) => {
+      const diff = later.since(earlier, { largestUnit });
+      it(`earlier.since(later, ${largestUnit}) == later.since(earlier, ${largestUnit}).negated()`, () =>
+        equal(`${earlier.since(later, { largestUnit })}`, `${diff.negated()}`));
+      it(`earlier.until(later, ${largestUnit}) == later.since(earlier, ${largestUnit})`, () =>
+        equal(`${earlier.until(later, { largestUnit })}`, `${diff}`));
+      it(`earlier.add(${diff}) == later`, () => assert(earlier.add(diff).equals(later)));
```

Comment:
Same. Adding `diff` to `earlier` isn't the same as subtracting `diff` from `later`. The latter is what (A) guarantees. The former is what (B) guarantees.